pkg/consts/consts.go

```diff
@@ -146,6 +150,13 @@ const (
 	// VMASCacheTTLDefaultInSeconds is the TTL of the vmas cache
 	VMASCacheTTLDefaultInSeconds = 600
 
+	// VmssFlexCacheTTLDefaultInSeconds is the TTL of the vmss flex cache
+	VmssFlexCacheTTLDefaultInSeconds = 600
+	// VmssFlexVMCacheTTLInSeconds is the TTL of the vmss flex vm cache
+	VmssFlexVMCacheTTLInSeconds = 600
+	// VmssFlexVMStatusCacheTTLInSeconds is the TTL of the vmss flex vm cache
```

Comment:
I guess it should be vmss flex vm "status" cache?

Reply:
I mean L155 and L157 seem to be the same comment "<var-name> is the TTL of the vmss flex vm cache". So I guess L157 should be "VmssFlexVMStatusCacheTTLInSeconds is the TTL of the vmss flex vm status cache"?

Reply:
Aah, good catch! Thanks for finding this!